### DISCUSSION TITLE
JDK recommended version and MSSQL driver

### DISCUSSION THREAD

**Discussion by mattgagliardi** [quarkusio/quarkus#25352] [Community]
Quarkus docs suggest a JDK of v11+. MSSQL JDBC driver (7.2.2) appears to be the JRE8 (rather than JRE11) JAR. Curious about this apparent discrepancy.

**Comment by maxandersen** [comment]
Quarkus requires minimum java 11 - if mysql driver works with java 8 too that is great but it does not change java 11 is required. 

Might be the docs where not updated. Feel free to open issue or even PR if using java 11 variation worked for you.

**Comment by mattgagliardi** [comment]
The JDBC driver for MSSQL (in Quarkus) is v7.2.2, which has two options; one for jre8 and one for jre11. It appears (unless I totally misread something, always possible) the one for jre8 was used at build time. 
So Quarkus has a JDK11 requirement but looks to be using a DB driver meant for jre8 when there is one available for jre11. Was just curious about the delta there. This link to the Quarkus Maven repo will help (link).
https://mvnrepository.com/artifact/io.quarkus/quarkus-jdbc-mssql/2.9.0.Final

Here’s the jre11 version of the 7.2.2 MSSQL driver. https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc/7.2.2.jre11
